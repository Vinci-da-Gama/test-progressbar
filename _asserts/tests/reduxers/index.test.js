import React from 'react';
import { shallow } from 'enzyme';

import RootReducer from '../../src/reduxers';

describe('In reduxers/index.js Test RootRedux: ', () => {
    it('should generate Root Reducer correctly.', () => {
        const wrapper = shallow(<RootReducer />);
        expect(wrapper).toMatchSnapshot();
    });
});
