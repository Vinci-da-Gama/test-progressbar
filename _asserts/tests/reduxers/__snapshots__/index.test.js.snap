// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`In reduxers/index.js Test RootRedux:  should generate Root Reducer correctly. 1`] = `<Component />`;
